pkgs/tools/wayland/gtklock/playerctl-module.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gnumake
+, pkg-config
+, gtk3
+, playerctl
+, libsoup
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gtklock-playerctl-module";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "jovanlanik";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-kzGgqFDTeKL6Pfjram7pqVcIm8Avxsvpn1qFrcpd8dw=";
+  };
+
+  nativeBuildInputs = [ gnumake pkg-config ];
```

`gnumake` being part of `stdenv`, this shouldn't be necessary.

```suggestion
  nativeBuildInputs = [ pkg-config ];
```